### PR TITLE
Label some Pikachu's Beach related routines

### PR DESCRIPTION
**Pull request by Sanqui**

### CODE REVIEW

File: engine/surfing_minigame.asm [hunk 1]
```diff
@@ -1018,73 +1018,73 @@ SurfingMinigame_TileInteraction:
...
-	ld [wc5e3 + 1], a
+	ld [wSurfingMinigamePikachuSpeed + 1], a
 	ld a, l
-	ld [wc5e3], a
+	ld [wSurfingMinigamePikachuSpeed], a
 	ret
 
-Subtract64Fromwc5e3:
-	ld a, [wc5e3 + 1]
+Subtract64FromwSurfingMinigamePikachuSpeed:
```
Comment (PikalaxALT) [inline engine/surfing_minigame.asm]: `SurfingMinigame_ReduceSpeedBy64`

File: engine/surfing_minigame.asm [hunk 2]
```diff
@@ -1018,73 +1018,73 @@ SurfingMinigame_TileInteraction:
...
-	ld [wc5e3 + 1], a
+	ld [wSurfingMinigamePikachuSpeed + 1], a
 	ld a, l
-	ld [wc5e3], a
+	ld [wSurfingMinigamePikachuSpeed], a
 	ret
 
-Subtract128Fromwc5e3:
-	ld a, [wc5e3 + 1]
+Subtract128FromwSurfingMinigamePikachuSpeed:
```
Comment (PikalaxALT) [inline engine/surfing_minigame.asm]: `SurfingMinigame_ReduceSpeedBy128`

File: engine/surfing_minigame.asm [hunk 3]
```diff
@@ -1181,9 +1181,9 @@ Func_f8778:
 	ret
 
 Func_f87a8:
```
Comment (PikalaxALT) [inline engine/surfing_minigame.asm]: `SurfingMinigame_GetSpeedDividedBy32`

**Comment by Sanqui** [comment]
Good catches, thanks.

**Pull request merged**